ProcessLib/SourceTerms/CreateNodalSourceTerm.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateNodalSourceTerm.h"
+
+#include "BaseLib/FileTools.h"
+#include "ProcessLib/Utils/ProcessUtils.h"
+#include "NodalSourceTerm.h"
+
+namespace ProcessLib
+{
+std::unique_ptr<NodalSourceTerm> createNodalSourceTerm(
+    BaseLib::ConfigTree const& config,
+    const NumLib::LocalToGlobalIndexMap& dof_table, std::size_t const mesh_id,
+    std::size_t const node_id, const int variable_id, const int component_id)
+{
+    DBUG("Constructing NodalSourceTerm from config.");
+    //! \ogs_file_param{prj__process_variables__process_variable__source_terms__source_term__type}
+    config.checkConfigParameter("type", "Nodal");
+
+    //! \ogs_file_param{prj__process_variables__process_variable__source_terms__source_term__value}
+    auto const nodal_value = config.getConfigParameter<double>("value");
```

Comment:
The `<value>` tag belongs to the "Nodal" type, so the `ogs_file_param` must be `..._source_term__Nodal__value`. And then you need a new folder `Nodal/` in the `Documentation/.../source_term/`. Compare to [DirichletBC.cpp](https://github.com/ufz/ogs/blob/92f2bf259feabdebdf666d0b1bab0e79620e611e/ProcessLib/BoundaryCondition/DirichletBoundaryCondition.cpp#L59-L63)